src/commands/syncModel/run/README.md

```diff
@@ -66,6 +66,35 @@ const params: SyncModelRunParams = {
 await syncModelRun(params);
 ```
 
+### Partial Sync
```

Comment:
We should probably incorporate it into the readme instead of having the `Partial sync` section since the partial sync is not sort of a default.